.github/workflows/release.yml

```diff
@@ -15,46 +15,64 @@ on:
         required: false
 
 jobs:
-  prepare_framework:
-    runs-on: macos-13-xlarge
-    name: "Create XCFramework"
+  bump-version:
+    runs-on: macos-15
     steps:
-      - uses: actions/checkout@v4
-
-      - uses: ruby/setup-ruby@v1
-        with:
-          bundler-cache: true
-
-      - name: "Download Fastlane Certificate"
-        run: bundle exec fastlane prepare_signed_xcframework
-        env:
-          APP_STORE_CONNECT_KEY_ID: ${{ secrets.APP_STORE_CONNECT_KEY_ID }}
-          APP_STORE_CONNECT_ISSUER_ID: ${{ secrets.APP_STORE_CONNECT_ISSUER_ID }}
-          APP_STORE_CONNECT_KEY: ${{ secrets.APP_STORE_CONNECT_KEY }}
-          FASTLANE_KEYCHAIN_PASSWORD: ${{ secrets.FASTLANE_KEYCHAIN_PASSWORD }}
-          MATCH_GIT_PRIVATE_KEY: ${{ secrets.MATCH_GIT_PRIVATE_KEY }}
-          MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
-          MATCH_USERNAME: ${{ secrets.MATCH_USERNAME }}
-        shell: sh
```

Comment:
`h`: I'm a bit confused. This PR renames the fastlane workflow to `prepare_xcframework_signing` for signing, but I don't see any invocations of `prepare_xcframework_signing` nor do I see anything signing the XCFramework during the release.yml. Am I missing something? If I'm not mistaken, this PR removes signing of the XCFramework or am I missing something?